ote/src/clj/ote/services/transit_changes.clj

```diff
@@ -112,6 +112,10 @@
        (when (authorization/admin? user)
          (http/transit-response (detection/hash-recalculations db))))
 
+  (GET "/transit-changes/reset-hash-calculation-status" {user :user :as request}
```

Comment:
With same effort this could be made RESTful if taking following aproach:

```suggestion
  (DELETE "/transit-changes/hash-calculation/" {user :user :as request}
       (when (authorization/admin? user)
         (http/transit-response (detection/hash-recalculations db))))
```

Reply:
Had a copy paste mistake, updated the snippet now :)

Reply:
Changed